client/src/VideoComponent.js

```diff
@@ -0,0 +1,194 @@
+// version6
+import React from "react";
+import axios from "axios";
+import ReactPlayer from "react-player"; // Import ReactPlayer
+import "./App.css";
+
+const VideoComponent = ({ video, onUpVote, onDownVote, onRemove }) => {
+  const handleDelete = async () => {
+    try {
+      // await axios.delete(`/videos/${video.id}`);
+      await axios.delete(
+        `https://back-end-full-stack-project-assessment.onrender.com/videos/${video.id}`
+      );
+      onRemove(video.id);
+    } catch (error) {
+      console.error("Error deleting video:", error);
+    }
+  };
+
+  return (
+    <div className="video-component">
+      <h3>{video.title}</h3>
+      {/* Replace the iframe tag with ReactPlayer */}
+      <ReactPlayer
```

Comment:
It's a good idea to use `ReactPlayer` component here, I've just noticed, it hasn't been installed. => https://www.npmjs.com/package/react-player
You have installed `axios` for instance, not this one though